app/controllers/api/v0/product_images_controller.rb

```diff
@@ -6,8 +6,8 @@ class ProductImagesController < Api::V0::BaseController
       respond_to :json
 
       def update_product_image
+        authorize! :update, Spree::Product
         product = Spree::Product.find(params[:product_id])
-        authorize! :update, @product
```

Comment:
We still need this one, too, I think. Otherwise I can change images of other people's products, right?

Reply:
Right!